model/ftn/wmesmfmd.ftn

```diff
@@ -2826,6 +3049,44 @@
         enddo
       endif
 !
+! 2.h Set ESMF import grid corner coordinates, only for coupled with mediator
+!
+      if (med_present) then 
+        call ESMF_GridAddCoord( impGrid, &
```

Comment:
Check to make sure we can't get to this section of code w a unstructured grid and we need to have a if GTYPE if statement

Reply:
I checked and we only get to this routine for the select GTYPES, so no need for a if GTYPE statement: 

 ```
     if ( (GTYPE.eq.RLGTYPE).or.(GTYPE.eq.CLGTYPE) ) then
         write(msg,'(a)') trim(cname)// &
           ': Creating import grid for Reg/Curvilinear Mode'
         call ESMF_LogWrite(trim(msg), ESMF_LOGMSG_INFO)
         call CreateImpGrid( gcomp, rc=rc )
         if (ESMF_LogFoundError(rc, PASSTHRU)) return
      elseif (GTYPE.eq.UNGTYPE) then
         write(msg,'(a)') trim(cname)// &
           ': Creating import mesh for Unstructured Mode'
         call ESMF_LogWrite(trim(msg), ESMF_LOGMSG_INFO)
         call CreateImpMesh( gcomp, rc=rc )
```

Reply:
Is this related with the adding corner coordinates? I am asking because there is no check in the code that adds corner coordinates.